### ISSUE TITLE
[Feature Request] Add more hotkeys

### ISSUE THREAD

**Issue by nsk7even** [mate-desktop/caja#681]
Specifically for file management actions between two panes, that are displayed side-by-side (via F3), I am missing some hotkeys.

The basic filesystem actions _copy_, _move_, _link_ to other pane would be greatly improved by getting hotkeys!

Additionally, some other actions (not related to the side-by-side pane view) may benefit as well from an hotkey - like e. g. _duplicate_, _compress_ a file.

But my concrete request is for the side-by-side pane view as I like it and use it heavily!

**Comment by Bill-MI** [comment]
Thumbs up to a great idea.  The 2-pane mode will be greatly enhanced.

**Comment by kn00tcn** [comment]
isnt ctrl+c + ctrl+v duplicate? compress a file wont work for those that removed the archive manager or other weird situations (maybe utilizing caja actions is applicable here)

does F3 view have a switch pane hotkey like how browsers have ctrl+tab? this seems like the most important key before making 'X to other pane'

**Comment by nsk7even** [comment]
Yes, F6 will toggle between the panes. When sorting files, one has to press the Ctrl+x, F6, Ctrl+v, F6 sequence all the times, therefore. Meanwhile s. o. gave me the great tip how to assign custom hotkeys: https://ubuntu-mate.community/t/caja-hotkey-for-move-file-to-other-pane/7716/10
This works well, but may stop working in future (see the link)!

For other hotkeys that rely on launching external binaries (e. g. the compress example), displaying an error message would be appropriate. In fact, the context menu entry has to handle these cases as well, so we aren't talking about new logic here, but only about another call method.

**Comment by kn00tcn** [comment]
it would be cool to have a hotkey manager that calls context menus themselves

do you not have your hand on the mouse when doing copy/cut commands? especially when selecting multiple files that arent grouped together, it's all hold ctrl + click click click ctrl+x, click to other pane or window, ctrl+v, click back to first, all this in a matter of hundreds of miliseconds, plus i can use my mouse thumb for keypad enter

**Comment by nsk7even** [comment]
The distribution of my hands on input interfaces always is a little bit different to "normal" users, because of multiple reasons. I won't explain these reasons now in detail, cause this will exceed this topic.

Just to give a short idea: I have three displays which means three times horizontal way to travel for the mouse. And no, I don't want to increase the mouse acceleration - in contrary I lowered it, cause otherwise it gets ridicilously fast and not controllable over this distance.
Another reason is the 10 finger blind typing system for renamings/new folders etc. when sorting files, which directly collides with using the mouse at all - cause then you have to switch your right hand from keyboard to mouse to keyboard to mouse ... etc. and at every switch the hand has to find it's position first....

Using my assigned hotkey for moving files to the other pane is aeons faster now, than moving the mouse anywhere - cause I'm done faster than my hand would have even reached the mouse. So this is kind of perfect now. But I don't know what I should expect from the lack of this ability in future releases.
